src/transformers/models/albert/modeling_flax_albert.py

```diff
@@ -173,7 +173,6 @@ def setup(self):
         self.LayerNorm = nn.LayerNorm(epsilon=self.config.layer_norm_eps, dtype=self.dtype)
         self.dropout = nn.Dropout(rate=self.config.hidden_dropout_prob)
 
-    # Copied from transformers.models.bert.modeling_flax_bert.FlaxBertEmbeddings.__call__
```

Comment:
@sanchit-gandhi I have to break the connection here as the same method in BERT as `attention_mask` as an argument but adding it here breaks a lot of tests.

Reply:
Good for me - the embeddings don't use an attention mask for Albert (it's instead masked in the attention), so fine to break the connection here!